blog/content/post/2024-11-04-qubes-os-summit-post.md

```diff
@@ -0,0 +1,226 @@
+---
+title: 'Qubes Os Summit 2024'
+abstract: 'The sixth edition of the Qubes OS Summit, organized by Invisible 
+Things Lab and 3mdeb, took place at the end of September'
```

Comment:
This abstract can be improved, but simply drop whole content into ChatGPT and asking it for summary.

Reply:
Your suggestion has been addressed.